lib/features/payment/payment_method.dart

```diff
@@ -76,15 +77,16 @@ const String bankSchema = '''
     "additionalProperties": false
   }''';
 
-const String momoSchema = '''
+const String momoSchema = r'''
   {
     "properties": {
       "accountNumber": {
         "type": "string",
         "title": "Phone number",
         "description": "Mobile money account number of the recipient",
-        "minLength": 12,
-        "maxLength": 12
+        "minLength": 13,
+        "maxLength": 13,
+        "pattern": "^\\+2547[0-9]{8}$"
```

Comment:
This is cool! Can we rely on the PFI always sending a `pattern` here?

Reply:
since this comes from `requiredPaymentDetails` in [PaymentMethod](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#paymentmethod) (which is a json schema), we can't rely on the pfi always sending a `pattern`. this is just what yellow card decided to do, but we should at least suggest other pfis do the same imo.